.devcontainer/devcontainer.json

```diff
@@ -70,10 +70,10 @@
 		// custom
 		"ghcr.io/devcontainers/features/sshd:1": {},
 		"ghcr.io/devcontainers/features/docker-outside-of-docker:1": {},
-		"ghcr.io/devcontainers/features/kubectl-helm-minikube:1": {},
+		"ghcr.io/devcontainers/features/kubectl-helm-minikube:1.2.1": {}, // 1.2.2 seems to be broken
 		"ghcr.io/devcontainers/features/azure-cli:1": {},
-		"ghcr.io/devcontainers-contrib/features/pre-commit:2": {},
-		"ghcr.io/marcozac/devcontainer-features/shellcheck:1": {},
+		"ghcr.io/devcontainers-extra/features/pre-commit:2": {},
+		"ghcr.io/froazin/devcontainer-features/shellcheck:0": {},
```

Comment:
Seems quite minimal on amount of usage. Sure this is good as a backing?

Reply:
I think it will work for now (and I did take a look at the install logic). Can go back to the other one when fixed, or just create our own feature if none of them prove stable.